### DISCUSSION TITLE
1000 plots with same key vs 10*100 plots with different keys

### DISCUSSION THREAD

**Discussion by Isanderthul** [Chia-Network/chia-blockchain#5083] [Q&A]
Does 1000 plots with the same private key have the same chance of winning as 10 sets of 100 plots with different keys?
is there any benefit to increasing the number of plots per private key?

**Comment by lvcivs** [comment]
As far as I understand it, the only thing that matters is your percentage of the netspace. So in both of your examples it will be exactly the same chance of winning. 

The only difference is that in the latter case it will be a huge hassle to keep track of all your keys...
